t/tars/discovery/tars.t

```diff
@@ -63,7 +63,8 @@ _EOC_
         schema.properties.full_fetch_interval.default = 3
 _EOC_
 
-    $block->set_value("extra_init_by_lua_start", $extra_init_by_lua_start);
+    $block->set_value("extra_init_by_lua", $extra_init_by_lua);
+    $block->set_value("extra_init_before_apisix_init");
```

Comment:
Could we modify the content in `$init_by_lua_block` (for example, reorder the statements) to make sure `extra_init_by_lua` work in this file?

Reply:
So that we don't need to introduce another flag like extra_init_by_lua_start or extra_init_before_apisix_init

Reply:
ok,let me try.